target/linux/ipq807x/files/arch/arm64/boot/dts/qcom/ipq8074-sxr80.dts

```diff
@@ -0,0 +1,18 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/* Copyright (c) 2021, Flole <flole@flole.de> */
+
+/dts-v1/;
+
+#include "ipq8074-sxk80.dtsi"
+
+/ {
+	model = "Netgear SXR80";
+	compatible = "netgear,sxr80", "qcom,ipq8074";
+};
+
+&wifi {
+	status = "okay";
+
+	qcom,ath11k-calibration-variant = "Netgear-SXR80";
```

Comment:
What board-id does the stock use?
I doubt its 0x255 that ath11k will default to

Reply:
We talked on the forum about this: https://forum.openwrt.org/t/adding-openwrt-support-for-xiaomi-ax3600/55049/3053?u=flole, they didn't bother to change it.

Reply:
Ok, it was a while ago